cmd/src/server.rs

```diff
@@ -361,7 +361,12 @@ impl TiKVServer {
     }
 
     fn init_engines(&mut self) {
-        let env = get_env(self.encryption_key_manager.clone(), None /*base_env*/).unwrap();
+        #[allow(unused_mut)]
+        let mut base_env = None;
+        if self.config.rocksdb.s3.enabled {
```

Comment:
can we merge the logic into `get_env`?

Reply:
Ok, right now get_env is defined in engine_rocks/encryption.rs

Reply:
> can we merge the logic into `get_env`?

Can you check if the new usage is right...